cfme/base/ui.py

```diff
@@ -62,3 +107,612 @@ class Dashboard(CFMENavigateStep):
 
     def step(self):
         self.view.navigation.select('Cloud Intel', 'Dashboard')
+
+
+class ServerView(ConfigurationView):
+    @View.nested
+    class server(Tab):  # noqa
+        TAB_NAME = "Server"
+
+    @View.nested
+    class authentication(Tab):  # noqa
+        TAB_NAME = "Authentication"
+
+    @View.nested
+    class workers(Tab):  # noqa
+        TAB_NAME = "Workers"
+
+    @View.nested
+    class customlogos(Tab):  # noqa
+        TAB_NAME = "Custom Logos"
+
+    @View.nested
+    class advanced(Tab):  # noqa
+        TAB_NAME = "Advanced"
+
+
+@navigator.register(Server)
+class Details(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Configuration')
+
+    def step(self):
+        self.view.accordions.settings.tree.click_path(
+            self.obj.zone.region.settings_string,
+            "Zones",
+            "Zone: {} (current)".format(self.obj.zone.description),
+            "Server: {} [{}] (current)".format(self.obj.name,
+                self.obj.sid))
+
+
+@navigator.register(Server, 'Server')
+class ServerDetails(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.server.select()
+
+
+@navigator.register(Server)
+class Authentication(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.authentication.select()
+
+
+@navigator.register(Server)
+class Workers(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.workers.select()
+
+
+@navigator.register(Server)
+class CustomLogos(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.customlogos.select()
+
+
+@navigator.register(Server)
+class Advanced(CFMENavigateStep):
+    VIEW = ServerView
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        self.view.advanced.select()
+
+
+class ServerDiagnosticsView(ConfigurationView):
+    @View.nested
+    class summary(Tab):  # noqa
+        TAB_NAME = "Summary"
+
+    @View.nested
+    class workers(Tab):  # noqa
+        TAB_NAME = "Workers"
+
+    @View.nested
+    class collectlogs(Tab):  # noqa
+        TAB_NAME = "Collect Logs"
+
+    @View.nested
+    class cfmelog(Tab):  # noqa
+        TAB_NAME = "CFME Log"
+
+    @View.nested
+    class auditlog(Tab):  # noqa
+        TAB_NAME = "Audit Log"
+
+    @View.nested
+    class productionlog(Tab):  # noqa
+        TAB_NAME = "Production Log"
+
+    @View.nested
+    class utilization(Tab):  # noqa
+        TAB_NAME = "Utilization"
+
+    @View.nested
+    class timelines(Tab):  # noqa
+        TAB_NAME = "Timelines"
+
+    configuration = Dropdown('Configuration')
```

Comment:
Good point, laziness is the answer. I didn't want to define all the views for all the classes as we have no automation using any of them just yet.